requirements.txt

```diff
@@ -1,5 +1,5 @@
 isodate
-jsonschema
+git+https://github.com/Julian/jsonschema@main#egg=jsonschema[format]
```

Comment:
we need to wait for alpha version to be released with the change
https://github.com/Julian/jsonschema/commit/1a7a935fd3e4dc11089b0a9f6ac247d66dc6b358

Reply:
sure, I used the `main` branch for now in order to be able to start the implementation :slightly_smiling_face:

Reply:
Switched to [4.0.0a6](https://pypi.org/project/jsonschema/4.0.0a6/)

Also dropped Python 3.6 support as in jsonschema

Reply:
@nezhar It looks like a stable release ([v4.0.1](https://pypi.org/project/jsonschema/4.0.1/)) is now available.